### PR TITLE
py-dijitso: add missing dependencies

### PR DESCRIPTION
**Pull request by MarcusCalhoun-Lopez**
###### Type(s)

- [X] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13
Xcode 9.4

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
LGTM!

**Comment by reneeotten** [comment]
Looking at the ```setup.py``` file, it actually declares dependencies on ```numpy``` and ```six``` as well. In addition, there is an updated version (2018.1.0) in case you're motivated to update at the same time.

File: python/py-dijitso/Portfile [hunk 1]
```diff
@@ -27,6 +27,10 @@ checksums           rmd160  211776f660dcc86c0af81044402bedd0f287ec73 \
 python.versions     36
 python.link_binaries_suffix
 
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools
+}
+
 if {${subport} ne ${name}} {
```
Comment (reneeotten) [inline python/py-dijitso/Portfile]: the two if statements are the same, you can just put the ```livecheck.type   none``` in the first one

**Comment by seanfarley** [comment]
Yep, LGTM

**Comment by MarcusCalhoun-Lopez** [comment]
I agree that the port should probably be updated, but py-dijitso seems to be part of a larger FEniCS project.
It is possible that all of them would need to be updated at once.

**Pull request merged**